modules/userId/eids.js

```diff
@@ -182,9 +182,13 @@ export function createEidsArray(bidRequestUserId) {
   let eids = [];
   for (const subModuleKey in bidRequestUserId) {
     if (bidRequestUserId.hasOwnProperty(subModuleKey)) {
-      const eid = createEidObject(bidRequestUserId[subModuleKey], subModuleKey);
-      if (eid) {
-        eids.push(eid);
+      if (subModuleKey === 'pubProvided') {
+        eids = eids.concat(bidRequestUserId['pubProvided']);
```

Comment:
should this use `eids.push()` instead of `concat` like code below uses?

Reply:
bidRequestUserId['pubProvided'] is an array of eids. Code below pushes single eid element

Reply:
ah true, good point. i forgot the pubProvidedIds were arrays still.